src/main/resources/hudson/tasks/junit/JUnitResultArchiver/help-checksName.html

```diff
@@ -1,4 +1,4 @@
 <div>
     If provided, and publishing checks enabled, the plugin will use this name when publishing results to corresponding
-    SCM hosting platforms. If not, a default of "Tests" will be used.
+    SCM hosting platforms. If not, a default comprising the enclosing blocks names will be used.
```

Comment:
how about? enclosing blocks is an implementation detail not meant for a non developers of this plugin
```suggestion
    SCM hosting platforms. If not, a default including the current stage / branch names will be used.
```